### ISSUE TITLE
Facing Issues while creating a Rule

### ISSUE THREAD

**Issue by abdulquddus** [johnitvn/yii2-rbac-plus#13]
Hi,
Sorry could be a silly question. This is my first time to use RBAC in any YII2 project. I want to ask is what is the class name in the RULEs. As what ever I give to it got every class does not exist.
![rules issue](https://cloud.githubusercontent.com/assets/11033422/10161170/9a948092-66bb-11e5-871f-9eef6c505b54.png)
Hope you won't mind of this silly question as I'm a learner. Thanks in advance.

**Comment by duynguyen0511** [comment]
You have to create class first before you can use it as rule . Read more ( http://www.yiiframework.com/doc-2.0/guide-security-authorization.html#using-rules)

**Comment by abdulquddus** [comment]
Thanks for your reply actually what I want to know is where should I create that class and how should I declare it in the class name of Rule Manager. Sorry again a silly question.

**Comment by Goancona** [comment]
Have you solvedd this? I have the same issue D:

**Comment by abdulquddus** [comment]
Yes I've been able to solve the problem by creating a folder in the backend named it rbac. Then I have added all the rules one by one in that folder. When tried to create the rule "backend\rbac\AuthorRule" given this in the class name without quotes.
